Makefile

```diff
@@ -2,26 +2,47 @@ HOSTNAME=github.com
 NAMESPACE=iterative
 NAME=iterative
 VERSION=0.0.${shell date +%s}+development
-OS_ARCH=${shell go env GOOS}_${shell go env GOARCH}
-BINARY=terraform-provider-${NAME}
-INSTALL_PATH=~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
+GOOS=${shell go env GOOS}
+GOARCH=${shell go env GOARCH}
+TF_PLUGIN_INSTALL_PATH=~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${GOOS}_${GOARCH}
+TPI_PATH ?= $(shell pwd)
+LEO_PATH ?= $(shell pwd)/cmd/leo
```

Comment:
If we're only using these paths to specify the build entry point, they can be relative to the Makefile. Actually, since we're only using them in one single location, there's not much point in defining constants for them